platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -484,8 +496,22 @@ static boolean handleMouseEvent(final MotionEvent event) {
 		final float tiltX = (float)-Math.sin(orientation) * tiltMult;
 		final float tiltY = (float)Math.cos(orientation) * tiltMult;
 
-		final float verticalFactor = event.getAxisValue(MotionEvent.AXIS_VSCROLL);
-		final float horizontalFactor = event.getAxisValue(MotionEvent.AXIS_HSCROLL);
+		float verticalFactor = 0;
+        float horizontalFactor = 0;
+        
+        // If event came from RotaryEncoder (Bezel or Crown rotate event on Wear OS smart watches),
+        // convert it to vertical mouse wheel event.
+        if (event.isFromSource(InputDevice.SOURCE_ROTARY_ENCODER)) {        	
+        	if (rotaryInputAxis.equals(AXIS_X)) {
+        		horizontalFactor = event.getAxisValue(MotionEvent.AXIS_SCROLL);        		
```

Comment:
The negation should be applied here as well.

Reply:
fixed